docs/aws.md

```diff
@@ -0,0 +1,67 @@
+# AWS
+
+## Configuration
+
+### Example
+
+<!-- UPDATE THIS JSON EXAMPLE -->
+
+```json
+{
+  "settings": [
+    {
+      "provider": "aws",
+      "domain": "domain.com",
+      "host": "@",
+      "ip_version": "ipv4",
+      "ipv6_suffix": "",
+      "aws_access_key_id": "ffffffffffffffffffff",
+      "aws_secret_access_key": "ffffffffffffffffffffffffffffffffffffffff",
+      "hosted_zone_id": "A30888735ZF12K83Z6F00",
+      "region": "eu-central-1",
+      "ttl": 60
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"`
+- `"host"` is your host and can be a subdomain or `"@"` or the wildcard `"*"`
+- `"aws_access_key_id"`
+- `"aws_secret_access_key"`
```

Comment:
I think we can remove the `aws_` prefix, it's obvious it's for aws/route53. I would also eventually rename those two to:



```suggestion
- `"key_id"`
- `"key"`
```

Reply:
I saw other comments about this as well.

My idea was to make it more familiar to AWS users, these names are well known by users and brainless when using a tool: "Oh... so this is the key and that is the secrets".